src/Diagnostics.DataProviders/DataProviders/GeoMasterDataProvider.cs

```diff
@@ -73,6 +73,27 @@ private IGeoMasterClient InitClient()
             return stickyToSlotSettings;
         }
 
+        public async Task<VnetValidationFailureDetails> VerifyHostingEnvironmentVnet(string subscriptionId, string vnetResourceGroup, string vnetName, string vnetSubnetName, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            var path = string.Format(@"subscriptions/{0}/providers/Microsoft.Web/verifyHostingEnvironmentVnet", subscriptionId);
```

Comment:
Do you want to use the SitePathUtility? Just to keep things consistent.

Reply:
I don't believe that function can be used directly here as this path has no site name in it...